File: app/src/main/java/com/panda/ap_anime/SplashActiviity.java
package com.panda.ap_anime;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import butterknife.BindView;
import butterknife.ButterKnife;
import carbon.widget.ImageView;
import com.panda.ap_anime.c_anmiations_utils.CAnmiation;
import com.panda.ap_anime.c_anmiations_utils.CAnmiationHelper;

class SplashActiviity extends AppCompatActivity {
    @BindView(R.id.main_image)
    ImageView imageView;
    CAnmiation cAnmiation;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        ButterKnife.bind(this);
        initView();
        DoSplash();
    }

    private void DoSplash() {
        cAnmiation.AppllyAnmiation(CAnmiationHelper.SHOW_VIEW_VISIBLE, imageView, () ->
        {
            startActivity(new Intent(this,MainActivity.class));
        });
    }
    private void initView()
    {
        cAnmiation = new CAnmiation(this,CAnmiation.FixedDureation);
    }
}

File: app/src/main/java/com/panda/ap_anime/SplashActiviity.kt
package com.panda.ap_anime

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import butterknife.BindView
import butterknife.ButterKnife
import carbon.widget.ImageView
import com.panda.ap_anime.c_anmiations_utils.CAnmiation
import com.panda.ap_anime.c_anmiations_utils.CAnmiationHelper

internal class SplashActiviity : AppCompatActivity() {
    @BindView(R.id.main_image)
    var imageView: ImageView? = null
   lateinit var cAnmiation: CAnmiation

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        ButterKnife.bind(this)
        initView()
        DoSplash()
    }

    private fun DoSplash()
    {
        startActivity(
            Intent(
                this,
                MainActivity::class.java
            )
        )
        finish()

    }

    private fun initView() {
        cAnmiation = CAnmiation(this, CAnmiation.FixedDureation)
    }
}
